### PR TITLE
Add CONTRIBUTING.md detailing release process

### PR DESCRIPTION
**Pull request by bclement-ocp**
Partially moved from docs/

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+This guide is intended to as best as possible reflect the current state of the processes and tools used in the development of Alt-Ergo. It may get stale or outdated from time to time: if you notice anything incorrect or that otherwise no longer applies in this guide, please signal it in an issue or fix it in a pull request. If you find it unclear or imprecise on certain points, please open an issue as well so that we can try to make it better.
+
+## Pull Request Guidelines
+
+External contributions to Alt-Ergo may be proposed using git's standard pull request mechanism from your `base` branch to our `target` one ( `next` by default).
+The following terms apply to all such contributions:
+
+* We require all pull requests to pass :
+
+	* Our non regression suite, you can run `make non-regression` to check if your changes affect the performance and answers of Alt-Ergo.
```
Comment (Halbaroth) [inline CONTRIBUTING.md]: I think the command is `make runtest`.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+### Run benchmarks
+
+Once the release branch has been confirmed, the RM runs some benchmarks to compare the state of the release branch against the last released version. These benchmarks are stored on an internal OCamlPro machine. The following benchmarks should be run (note: the Zulip bot can help):
+
+ - ae-format benchmarks (internal dataset in the native format)
+ - ae-format-fpa benchmarks (internal dataset in the native format, using the floating-point theories)
+ - benchtop-tests benchmarks (smtcomp subset)
+
+If regressions are found, they must be first confirmed by checking that they are still present in a different environment, to ensure they are not spurious regressions due to the weak table. The simplest way to do it is to compile with a different version of OCaml (this is what the Zulip bot does).
```
Comment (Halbaroth) [inline CONTRIBUTING.md]: We could add a link to the PR that explains the weak table issue.

**Review by Halbaroth — APPROVED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (bclement-ocp) [reply CONTRIBUTING.md]: Possibly. This is imported from the existing sphinx doc, I admit I did not proofread it :)

(Should it be `make runtest-ci` actually?)

File: CONTRIBUTING.md (hunk 2 above)
Comment (bclement-ocp) [reply CONTRIBUTING.md]: You mean #563 ? Good idea, will do

File: CONTRIBUTING.md (hunk 1 above)
Comment (Halbaroth) [reply CONTRIBUTING.md]: Don't worry, I know it was an old documentation but since I read it completely ;)
I think `make runtest-ci` is for the CI only.

Comment (bclement-ocp) [reply CONTRIBUTING.md]: `runtest-ci` just runs more tests than `runtest` (or different tests?); the doc is stating "this is what we require to merge", and we do require `runtest-ci` to pass before merging

**Pull request merged**